zndraw/server/events.py

```diff
@@ -9,8 +10,33 @@
 from flask import request, session
 from flask_socketio import call, emit, join_room
 
+from zndraw.utils import typecast
+
 from ..app import cache
 from ..app import socketio as io
+from .data import (
```

Comment:
At this point, I'd call the file not `data.py` but `data_transfer_objects.py` and do from `.data_transfer_objects import *`

Reply:
The number of imports will be dramatically reduced in the future, so I'd like to keep this - seeing the number of imports being reduced also acts as motivation.

Also, I don't like wildcard imports.